### PR TITLE
Add project tag

### LINKED ISSUE #235: Tag all items created in H3 with an H3 project tag
All new items created in H3 should be tagged with the Project tag "H3," similar to how all H2 items are tagged with H2.

Also, all items migrated to H3 from H2 should have an H3 tag added to them. Any existing Hydrus or H2 tags should be retained (this likely will need to be a separate ticket to handle during content migration).

### PR DESCRIPTION
**Pull request by lwrubel**
Fixes #235 to add an "H3" project tag to collections and works created in H3. 

Screenshot 2024-12-20 at 11 26 41 AM

Screenshot 2024-12-20 at 12 40 51 PM

### CODE REVIEW

File: app/services/to_cocina/collection/mapper.rb [hunk 1]
```diff
@@ -40,6 +40,12 @@ def params
           administrative: { hasAdminPolicy: Settings.apo }
         }.compact
       end
+
+      def request_params
+        params.tap do |params_hash|
+          params_hash[:administrative] = { hasAdminPolicy: Settings.apo, partOfProject: Settings.project_tag }
```
Comment (justinlittman) [inline app/services/to_cocina/collection/mapper.rb:46]:
```
params_hash[:administrative][:partOfProject] = Settings.project_tag
````
?

File: app/services/to_cocina/work/mapper.rb [hunk 2]
```diff
@@ -48,6 +48,12 @@ def access
         { view: 'world', download: 'world', license: work_form.license.presence,
           useAndReproductionStatement: I18n.t('works.edit.fields.license.terms_of_use') }.compact
       end
+
+      def request_params
+        params.tap do |params_hash|
+          params_hash[:administrative] = { hasAdminPolicy: Settings.apo, partOfProject: Settings.project_tag }
```
Comment (justinlittman) [inline app/services/to_cocina/work/mapper.rb:54]: Same suggestion as above.

**Review by justinlittman — APPROVED** [review]
Approve pending consideration of suggestions.

File: app/services/to_cocina/collection/mapper.rb (hunk 1 above)
Comment (lwrubel) [reply app/services/to_cocina/collection/mapper.rb]: Yep, that works. Making that change.

**Pull request merged**